lib/Borel/Error.hs

```diff
@@ -0,0 +1,54 @@
+{-# LANGUAGE OverloadedStrings #-}
+-- | Copyright 2013-2015 Anchor Systems, Pty Ltd and Others
```

Comment:
We should probably do the license header thing consistently or not at all. I think they're annoying but apparently Debian developers think they're useful or something. @afcowie thoughts? Maybe we could add an sdist step to add them pre-build, so we don't have to keep them in version control?

In any case, I'm pretty sure this codebase didn't exist in 2013.